sdjournal/journal.go

```diff
@@ -57,6 +57,11 @@ const (
 	SD_JOURNAL_INVALIDATE = int(C.SD_JOURNAL_INVALIDATE)
 )
 
+// The maximum value for a time.Duration.  Used for sdjournal.WaitIndefinitely()
+const (
+	maxDuration = "2540400h10m10.000000000s"
```

Comment:
Make this public, it's part of the Wait() api.  I'd rather it be like what @jonboulle suggested, IndefiniteWait, or InfiniteDuration... something like that.